source/direct_debit/index.html.md.erb

```diff
@@ -204,7 +204,7 @@ The query parameters you can use to search are:
 - `status`
 - `mandate_id`
 
-All the parameters are case insensitive. You can search for partial values of `reference`.
+All the parameters are case insensitive but must be an exact match.
```

Comment:
Once again, I think `state` is case sensitive.

Reply:
Also fixed in https://github.com/alphagov/pay-tech-docs/pull/320/commits/ec5793bfcb5ccd6af48b5f7977c3e98505ec0253.